pkg/apis/kudo/v1beta1/test_types.go

```diff
@@ -90,7 +90,7 @@ type TestAssert struct {
 type ObjectReference struct {
 	corev1.ObjectReference `json:",inline"`
 	// Labels to match on.
-	Labels map[string]string
+	Labels map[string]string `json:""`
```

Comment:
Let's be explicit here
```suggestion
	Labels map[string]string `json:"labels"`
```